instructure/canvas-ios
# Add pronouns to assignment overrides

refs: MBL-13763
affects: teacher
release note: none

Test plan:
* Teacher app > narmstrong : t
* CS 1400 > Assignments > MGP Assignment 3 > Edit
* Tap `+ Add Due Date` > Assignees
* Tap `+ Add Assignee`
* Student1 Armstrong should have pronouns displayed
* Tap Student1 Armstrong
* Pronouns should be displayed again
* Tap Done
* View Due Dates from assignment details
* Student1 should have pronouns
